universe/Conditions.cpp

```diff
@@ -1501,14 +1501,14 @@ unsigned int Target::GetCheckSum() const {
 // Homeworld                                             //
 ///////////////////////////////////////////////////////////
 Homeworld::Homeworld() :
-    Condition(),
-    m_names()
+    Homeworld(std::vector<std::unique_ptr<ValueRef::ValueRef<std::string>>>{})
```

Comment:
AFAIK `{}` should be sufficient here.

Reply:
I thought the same, but it can't resolve the overload with the move constructor:
````
1>  c:\users\geoff\desktop\fosdk11\freeorionalt2\universe\conditions.cpp(1504): error C2668: 'Condition::Homeworld::Homeworld': ambiguous call to overloaded function
1>  c:\users\geoff\desktop\fosdk11\freeorionalt2\universe\conditions.h(250): note: could be 'Condition::Homeworld::Homeworld(Condition::Homeworld &&)'
1>  c:\users\geoff\desktop\fosdk11\freeorionalt2\universe\conditions.h(234): note: or       'Condition::Homeworld::Homeworld(std::vector<std::unique_ptr<ValueRef::ValueRef<std::string>,std::default_delete<_Ty>>,std::allocator<std::unique_ptr<_Ty,std::default_delete<_Ty>>>> &&)'
1>          with
1>          [
1>              _Ty=ValueRef::ValueRef<std::string>
1>          ]
1>  c:\users\geoff\desktop\fosdk11\freeorionalt2\universe\conditions.cpp(1505): note: while trying to match the argument list '(initializer list)'
````